app/main.py

```diff
@@ -1,6 +1,20 @@
+import os
+
+import requests
+
+
+API_KEY = os.getenv("API_KEY")
+
+FILTER = "Paris"
+URL = "https://api.weatherapi.com/v1/current.json"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    params = {"key": API_KEY, "q": FILTER}
+
+    response = requests.get(URL, params=params)
+
+    print(response.content)
```

Comment:
emmm .. it's raw json,)) I guess the task implies that we should make some formatting)

Reply:
Idk, I think its ok to return raw json with full data, depends on what we gonna get from it.